app/reducers/dex.js

```diff
@@ -480,6 +481,11 @@ export default function ln(state = {}, action) {
         alreadyPaid: false,
         config: null
       };
+    case RESET_DEXACCOUNT:
+      return {
+        ...state,
+        dexAccount: null
+      };
```

Comment:
Should be updated `dexAccount` not only in this reducer but in `walletLoader` since the selector reads from there: https://github.com/decred/decrediton/blob/ee04c5d79dafd9323a2945d0fc8eda8b896248e9/app/selectors.js#L1961

Without this change, the user needs to restart the app (or reopen the wallet) to reach the dex wallet creation page.

Reply:
Whoops good call.